src/app.py

```diff
@@ -1,40 +1,64 @@
-import logging
+import logging, coloredlogs, yaml, os, sys, urllib3, requests
 import os
-
 import click
 
-from vcf_file_generator import VcfFileGenerator
-from orthology_file_generator import OrthologyFileGenerator
-from daf_file_generator import DafFileGenerator
-from expression_file_generator import ExpressionFileGenerator
+from generators import *
```

Comment:
Also a no-no is the use of `import *` - best to explicitly name the variables you want to bring in, or alias the module if you want brevity (e.g: `import generators as g`)